src/dotnet/APIView/APIViewWeb/Client/css/site.scss

```diff
@@ -26,6 +26,10 @@ a.navbar-brand
     word-break: break-all;
 }
 
+tr:nth-child(even) {
```

Comment:
unrelated?

Reply:
https://getbootstrap.com/docs/4.0/content/tables/#striped-rows

Reply:
nope. Included one  of the suggestion from Jonathan to show striped table.

Reply:
Bootstrap can do this in a simpler way. I'd suggest reverting this and I'll create a PR that demonstrates a few style changes I would propose.

Reply:
I've just quickly done a few of my suggested 'easy wins' here: https://github.com/Azure/azure-sdk-tools/pull/1751
Feel free to take the changes and apply them into a proper PR if you want.

Reply:
The striped tables get messed up when using the search filter:
![image](https://user-images.githubusercontent.com/54595583/123854206-46a74b00-d8d3-11eb-8a35-97c2d4faa940.png)